ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -884,6 +899,23 @@ literals:
 - valve_status:
   - OPEN
   - CLOSED
+- zone_control_cooling_command:
+  - COOLING
+  - OFF
+- low_battery_alarm:
+  - ACTIVE
+  - INACTIVE
+- load_bypass_mode:
+  - OFF
+  - BYPASS
+- gas_release_status:
+  - OPEN
+  - CLOSED
+# apparent_power_sensor (and reactive_power_sensor) has error on units and can't be resolved
```

Comment:
open an issue if you continue having this error, we do not want to support apparentpower or reactivepower